core/file_walker.go

```diff
@@ -51,26 +51,26 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 }
 
 func getSubTreeConcurrently(path string, parent *File, readDir ReadDir, ignoredFolders map[string]struct{}, c chan bool, mutex *sync.Mutex, wg *sync.WaitGroup) *File {
-	ret := &File{}
+	result := &File{}
```

Comment:
I'm not too fussed (my team uses `result` but I can work with both). But if we decide to change convention, we need to do it in the whole codebase in the name of consistency. If you'd like to use `ret` instead of `return`, please create MR where you change it everywhere.